src/main/java/com/adyen/model/marketpay/CreateAccountHolderRequest.java

```diff
@@ -32,87 +37,82 @@
  * CreateAccountHolderRequest
  */
 public class CreateAccountHolderRequest {
-    @SerializedName("createDefaultAccount")
-    private Boolean createDefaultAccount = null;
-
-    @SerializedName("description")
-    private String description = null;
-
     @SerializedName("accountHolderCode")
     private String accountHolderCode = null;
 
     @SerializedName("accountHolderDetails")
     private AccountHolderDetails accountHolderDetails = null;
 
-    @SerializedName("processingTier")
```

Comment:
Do we still need to add this `@SerializedName` annotation? Can you test that?

Reply:
I just tried the unit tests for this endpoint without the annotation and it still passed. I guess that means we can leave it out?